common/lib/xmodule/xmodule/partitions/partitions_service.py

```diff
@@ -3,41 +3,114 @@
 user partitions.  It uses the user_service key/value store provided by the LMS runtime to
 persist the assignments.
 """
-from abc import ABCMeta, abstractproperty
+from django.conf import settings
```

Comment:
Yes, it's Ok.  Just as modulestore/django.py is django-dependent, we can say that the partitions folder is also django-dependent.